gandalf.js

```diff
@@ -1 +1,2 @@
 hui hui
```

Comment:
There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?